webrtc_dsp/rtc_base/platform_thread_types.cc

```diff
@@ -29,6 +31,10 @@ PlatformThreadId CurrentThreadId() {
   return zx_thread_self();
 #elif defined(WEBRTC_LINUX)
   return syscall(__NR_gettid);
+#elif defined(WEBRTC_FREEBSD)
```

Comment:
is this freebsd-specific? can't we use something like WEBRTC_UNIX or WEBRTC_POSIX?

Reply:
Yes, this is OS specific. There is no POSIX way to get thread id.